purchases/src/main/kotlin/com/revenuecat/purchases/Purchases.kt

```diff
@@ -410,6 +416,13 @@ class Purchases internal constructor(
 
     /**
      * Purchases a [StoreProduct]. If purchasing a subscription, it will choose the default [SubscriptionOption].
+     *
+     * The default [SubscriptionOption] logic:
+     *   - Filters out offers with "rc-ignore-default-offer" tag
+     *   - Uses longest free trial (if one exists)
+     *   - Uses first phase with lowest price (if one exists)
+     *   - Falls back to use base plan
```

Comment:
I'd link back to the other method

Reply:
Can that be done with docs??? 🤔 I don't think it would show the explanation when hovering over the method in Android Studio then

Reply:
Was just thinking of a @see tag referencing the other method https://docs.oracle.com/javase/7/docs/technotes/tools/windows/javadoc.html#see but this is fine too... up to you
Otherwise looks great

Reply:
Oh, TIL!  It doesn't look like we use that right now in the SDK so I'll stay with what is there right now but this is good to know if we do any doc cleanup in the future 😇